Packs/Base/Scripts/DBotSuggestClassifierMapping/DBotSuggestClassifierMapping.py

```diff
@@ -526,8 +533,25 @@ def get_alias_index(field_name, alias):
     return SIEM_FIELDS[field_name]['aliases'].index(alias)  # type: ignore
 
 
-def get_most_relevant(field_name, field_mappings):
-    candidates = sorted(field_mappings, key=lambda x: get_alias_index(field_name, x[1]))
+def get_most_relevant_json_field(field_name, json_field_to_alias):
+    if not json_field_to_alias:
+        return
+
+    # calculate jaccard score for each alias, and get the candidates with max score
+    scores = {}
+    for json_field, alias in json_field_to_alias.items():
+        scores[json_field] = jaccard_similarity_for_string_terms(json_field, alias)
+    scores = {k: v for k, v in scores.items() if v == max(scores.values())}
+
+    # calculate jaccard score for each field, and get the candidates with max score
+    for json_field, alias in json_field_to_alias.items():
+        scores[json_field] = jaccard_similarity_for_string_terms(json_field, field_name)
+    scores = {k: v for k, v in scores.items() if v == max(scores.values())}
+
+    # for candidates with the same score with the least alias index
+    candidates = sorted(list(scores.keys()),
+                        key=lambda json_field: get_alias_index(field_name, json_field_to_alias[json_field]))
```

Comment:
Is it still going to work if `json_field` is not in `SIEM_FIELDS[field_name]`? might cause an exception?

Reply:
No, it will suppose to be there cuz probably it still match on some alias
anyway index return -1 and it's just for sorting so no error